test/OpenBCIConstants-test.js

```diff
@@ -636,7 +636,7 @@ describe('OpenBCIConstants', function() {
         //'channel 1, power on, gain 24, inputType normal, bias include, srb2 connect, srb1 dissconnect'
         describe('channel input selection works', function() {
             //this.timeout(5000);
-            it('channel 2', function() {
+            it('channel 2', function(done) {
```

Comment:
the mocha framework tests async functions with the done argument... you call it when the thing you are testing finishes, it's essentially tells the `it` to wait till is gets called.